packages/@sanity/vision/src/util/calendarDate.js

```diff
@@ -2,7 +2,7 @@ import moment from 'moment'
 
 function calendarDate(date) {
   return moment(date).calendar(null, {
-    sameElse: 'YYYY-MM-DD HH:mm:ss'
```

Comment:
This is also using moment, is this format correct here?

Reply:
will leave all formats as they are where moment is being used 👍